WordPressVIPMinimum/ruleset-test.inc

```diff
@@ -399,7 +399,7 @@ is_multi_author(); // Warning.
 the_sub_field( 'field' ); // Warning.
 the_field( 'field' ); // Warning.
 wp_remote_get( $url ); // Warning.
-setcookie( 'cookie[three]', 'cookiethree' ); // Warning.
+setcookie( 'cookie[three]', 'cookiethree' ); // Error.
```

Comment:
Do we need to update `ruleset-test.php` to account for it being an error?

https://github.com/Automattic/VIP-Coding-Standards/blob/71b5268a7b936cdebe5b3ecc7a1376e1cab19fdc/WordPress-VIP-Go/ruleset-test.php#L213